erpnext/manufacturing/doctype/production_plan/production_plan.py

```diff
@@ -205,6 +204,7 @@ def add_items(self, items):
 				'planned_start_date': now_datetime(),
 				'product_bundle_item': data.parent_item
 			})
+			pi._set_defaults()
```

Comment:
The change is acceptable, I am just wondering why this isn't handled at the framework level already. I never got time to investigate fully, hence unmerged. 😅

Reply:
Also `_set_default` is not used anywhere in ERPNext because as per convention `_method` is supposed to be internal and can be changed without any warning.